### PR TITLE
feat(spinbot): automated cherrypicks

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: spinbot/event/pull_request_cherry_pick_event_handler.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from gh import AddLabel, RemoveLabel, ParseCommitMessage
+from .handler import Handler
+from .command import GetCommands
+from .pull_request_event import GetPullRequest, GetRepo
+
+invalid_command_format = ("You must specify exactly 1 release to cherry-pick " +
```
Comment (jtk54) [inline spinbot/event/pull_request_cherry_pick_event_handler.py]: Would it make sense to extend this to a list? We sometimes pick changes into several versions.

Comment (lwander) [reply spinbot/event/pull_request_cherry_pick_event_handler.py]: My thinking was you just write:
> @spinnakerbot cherry-pick 1.10
> @spinnakerbot cherry-pick 1.9
> ...
WDYT?

Comment (jtk54) [reply spinbot/event/pull_request_cherry_pick_event_handler.py]: Seems OK. If this gets annoying we can change later.

Comment (lwander) [reply spinbot/event/pull_request_cherry_pick_event_handler.py]: btw that can all be done in 1 comment

**Pull request merged**